code/modules/surgery/organs/tongue.dm

```diff
@@ -18,7 +18,8 @@
 		/datum/language/ratvar,
 		/datum/language/aphasia,
 		/datum/language/piratespeak,
-		/datum/language/rlyehian,
+		/datum/language/rlyehian
+		/datum/language/apidite,
```

Comment:
```suggestion
		/datum/language/rlyehian,
		/datum/language/apidite,
```

Reply:
Yeah, I read through the Travis log and was fixing that.